typed-racket-lib/typed-racket/utils/tc-utils.rkt

```diff
@@ -280,8 +288,28 @@ don't depend on any other portion of the system
           (current-continuation-marks))))
 
 ;; are we currently expanding in a typed module (or top-level form)?
+;; (box/c (or/c #t #f))
 (define typed-context? (box #f))
 
+;; (box/c type-enforcement-mode?)
+(define type-enforcement-mode (box #f))
```

Comment:
It looks to me that `type-enforcement-mode` should be a parameter.

Reply:
Should `typed-context?` be a parameter too? Can it be?
Unless we know it's a yes to both, I'd rather leave this to do after merging.

Reply:
[`language-info.rkt`](https://github.com/racket/typed-racket/blob/master/typed-racket-lib/typed-racket/language-info.rkt) seems like a no-go for parameters